gateway-ha/src/main/java/io/trino/gateway/ha/module/HaGatewayProviderModule.java

```diff
@@ -14,14 +14,18 @@
 package io.trino.gateway.ha.module;
 
 import com.codahale.metrics.Meter;
+import com.google.inject.AbstractModule;
 import com.google.inject.Provides;
 import com.google.inject.Singleton;
 import io.dropwizard.auth.AuthFilter;
 import io.dropwizard.auth.Authorizer;
 import io.dropwizard.auth.basic.BasicCredentialAuthFilter;
 import io.dropwizard.auth.chained.ChainedAuthFilter;
+import io.dropwizard.core.server.DefaultServerFactory;
```

Comment:
@wendigo .. you are the jetty master at this stage .. could we use some Jetty classes directly for this already here?

Reply:
Fine to be done in a next step too..